RtspServer/Classes/SourceMedia.cs

```diff
@@ -73,7 +73,7 @@ public enum StreamState
         internal string m_Name;
         internal Uri m_Source;
         internal NetworkCredential m_SourceCred;
-        internal List<string> m_Aliases = new List<string>();
+        internal List<string> m_Aliases = new();
```

Comment:
Wonder if it makes more sense to use a HashSet, the order of the Aliases shouldn't be that important and it would reduce the search time.